integration/raft/migration_test.go

```diff
@@ -82,191 +77,142 @@ var _ = Describe("ConsensusTypeMigration", func() {
 	// suite will need to be completed and revised.
 	Describe("Raft to future-type migration", func() {
 		var (
-			orderer                        *nwo.Orderer
-			peer                           *nwo.Peer
-			syschannel, channel1, channel2 string
+			orderer            *nwo.Orderer
+			peer               *nwo.Peer
+			channel1, channel2 string
 		)
 
 		BeforeEach(func() {
-			network = nwo.New(raftMultiChannel(), testDir, client, StartPort(), components)
+			network = nwo.New(nwo.MultiChannelEtcdRaftNoSysChan(), testDir, client, StartPort(), components)
 			network.GenerateConfigTree()
 			network.Bootstrap()
 
 			orderer = network.Orderer("orderer")
 			peer = network.Peer("Org1", "peer0")
 
-			syschannel = network.SystemChannel.Name
-			channel1 = "testchannel1"
+			channel1 = "testchannel"
 			channel2 = "testchannel2"
 
 			o1Runner = network.OrdererRunner(orderer)
-
 			o1Proc = ifrit.Invoke(o1Runner)
 			Eventually(o1Proc.Ready(), network.EventuallyTimeout).Should(BeClosed())
-			Eventually(o1Runner.Err(), network.EventuallyTimeout, time.Second).Should(gbytes.Say(
-				fmt.Sprintf("Raft leader changed: 0 -> 1 channel=%s node=1", syschannel)))
 
-			By("Create & join first channel, deploy and invoke chaincode")
-			network.CreateChannel(channel1, orderer, peer)
-			Eventually(o1Runner.Err(), network.EventuallyTimeout, time.Second).Should(gbytes.Say(
-				fmt.Sprintf("Raft leader changed: 0 -> 1 channel=%s node=1", channel1)))
+			By("Create & join first channel")
+			channelparticipation.JoinOrdererAppChannel(network, channel1, orderer, o1Runner)
 		})
 
-		// This test executes the "green path" migration config updates on an etcdraft based system with a system channel
-		// and a standard channel, and verifies that these config updates have the desired effect.
+		// This test executes the "green path" migration config updates on an etcdraft based system, on a standard
+		// channel, and verifies that these config updates have the desired effect.
 		//
 		// The green path is entering maintenance mode, and then changing the consensus type.
-		// In maintenance mode we check that channel creation is blocked and that normal transactions are blocked.
+		// In maintenance mode we check that normal transactions are blocked.
```

Comment:
Earlier channel creation was blocked, now it is allowed in maintenance mode. Why it is allowed now, can you help me to understand it.

Reply:
Before, channel creation was done via a TX on the system channel. When the system channel was in maintenance mode, channel creation was not allowed. When there is no system channel, this check becomes moot.

Note that "maintenance mode" is the state of a channel, not an orderer. Thus, for example, when channel X on orderer o1 is in maintenance mode, this has no effect on any other channel that o1 is part of, and has no effect on the ability of o1 to join channel Y. It just blocks channel X to normal TXs and enforces certain config transition rules that facilitate consensus-type migration.

Reply:
Now. I am able to follow it. Thank you